services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User

```

Comment:
According to the checklist, use `get_user_model()` instead of `User`. It is the best practice.